docs/resource/index.md

```diff
@@ -1,24 +1,34 @@
 # SyliusResourceBundle
 
-There are plenty of things you need to handle for every single Resource in your web application.
+The **Sylius Resource Bundle** provides a powerful and extensible foundation for exposing your **business resources** (entities, aggregates, etc.) in a declarative way.  
+Rather than generating controllers or relying on rigid admin generators, it offers a flexible architecture that lets you focus on your domain model while the bundle handles the boilerplate.
 
-Several "Admin Generators" are available for Symfony, but we needed something really simple, that will allow us to have reusable controllers
-but preserve the performance and standard Symfony workflow. We did not want to generate any code or write "Admin" class definitions in PHP.
-The big goal was to have exactly the same workflow as with writing controllers manually but without actually creating them!
+A *Resource* is any business object you want to expose — for example, a `Product`, `Order`, or `UserProfile`.  
+You declare a resource by implementing the `ResourceInterface` and annotating it with the `#[AsResource(...)]` attribute.  
```

Comment:
I would remove this personally, or place it closer to the first example of a resource declaration